### PR TITLE
Add Chinese Translate

### PR DESCRIPTION
**Pull request by Bill-Haku**
Translation includes: 
- Diluc & Fischl's new costumes in `items.json5`
- Translate all in `locations.json5`
- Translate all in `organizations.json5`
- Translate Archon Quests in `quests.json5`
- Translate all in `sereniteapot.json5`
- Translate all in `system.json5`

Add new includes:
- Complete fishes in `items.json5`

### CODE REVIEW

**Comment by Bill-Haku** [comment]
@xicri PR of this week is ready. You can review it in your convenience.

File: dataset/dictionary/locations.json5 [hunk 1]
```diff
@@ -281,111 +283,134 @@
...
   },
   {
     en: "Chihu Rock",
     ja: "チ虎岩",
+    zhCN: "吃虎岩 / 螭虎岩",
+    // Note: In game the place is called 吃虎岩, however, there's the fact that the place
+    // was called 螭虎岩 in history and it was simplified to 吃虎岩 by the public over time
+    // in the setting of the game. Hence somtimes 螭虎岩 also can be seen in secondary creations.
+    // If you add a note for this word, I hope you can append this Chinese explaination at the same time:
+    // 此地曾名为螭虎岩，但随着时间流逝百姓将此地简化为了吃虎岩。特此说明。
```
Comment (xicri) [inline dataset/dictionary/locations.json5:312]:
```suggestion
    notes: "中国語の「螭虎岩」は古い時代の表記で、現在は簡略化されて「吃虎岩」と表記されるようになったという設定。 / 此地曾名为螭虎岩，但随着时间流逝百姓将此地简化为了吃虎岩。特此说明。"
```

File: dataset/dictionary/organizations.json5 [hunk 2]
```diff
@@ -137,62 +146,79 @@
...
     en: "Heyu Tea House",
     ja: "和裕茶館",
+    zhCN: "和裕茶馆",
+    tags: [ "liyue", "organization" ],
+  },
+  // Note: Recommend add this: (appeared mainly in stories about Yelan)
+  {
+    en: "Yanshang Teahouse",
+    ja: "岩上茶室",
+    zhCN: "岩上茶室",
```
Comment (xicri) [inline dataset/dictionary/organizations.json5:221]: Thanks, let's add it.

```suggestion
  {
    en: "Yanshang Teahouse",
    ja: "岩上茶室",
    pronunciationJa: "がんじょうちゃしつ",
    zhCN: "岩上茶室",
```

File: dataset/dictionary/quests.json5 [hunk 3]
```diff
@@ -91,21 +91,25 @@
   },
 
   // Inazuma
+  // Note: missing chapter 2 Prologue (zhCN: "振袖秋风问红叶",)
```
Comment (xicri) [inline dataset/dictionary/quests.json5:94]: Oops, I didn't add it.

```suggestion
  {
    en: "Autumn Winds, Scarlet Leaves",
    ja: "秋風に舞いし紅葉",
    zhCN: "振袖秋风问红叶",
    pronunciationJa: "あきかぜにまいしもみじ",
    tags: [ "inazuma", "quest-archon" ],
  },
```

**Review by xicri — APPROVED** [review]
Thanks, I have finished the review.

Regarding to quest names, I don't add world quests and commission quests so actively recently. If you are not interested in adding those translations, you don't have to add them.

File: dataset/dictionary/locations.json5 [hunk 4]
```diff
@@ -247,111 +283,130 @@
...
     ja: "月海亭",
+    zhCN: "月海亭",
     pronunciationJa: "げつかいてい",
     tags: [ "liyue", "location" ],
   },
   {
     en: "Chihu Rock",
     ja: "チ虎岩",
+    zhCN: "吃虎岩 / 螭虎岩",
+    notes: "中国語の「螭虎岩」は古い時代の表記で、現在は簡略化されて「吃虎岩」と表記されるようになったという設定。 / 此地曾名为螭虎岩，但随着时间流逝百姓将此地简化为了吃虎岩。特此说明。"
```
Comment (xicri) [inline dataset/dictionary/locations.json5:308]: Oops, I forgot the trailing comma.

```suggestion
    notes: "中国語の「螭虎岩」は古い時代の表記で、現在は簡略化されて「吃虎岩」と表記されるようになったという設定。 / 此地曾名为螭虎岩，但随着时间流逝百姓将此地简化为了吃虎岩。特此说明。",
```

**Review by xicri — CHANGES_REQUESTED** [review]

**Comment by Bill-Haku** [comment]
> Regarding to quest names, I don't add world quests and commission quests so actively recently. If you are not interested in adding those translations, you don't have to add them.

Sorry for the mistake that I only translated Archon Quests in `quests.json5`, which has been updated. Also, I will not translate the world quests temporarily.

**Comment by xicri** [comment]
@Bill-Haku

> Sorry for the mistake that I only translated Archon Quests in `quests.json5`, which has been updated. Also, I will not translate the world quests temporarily.

Oh, sorry for the confusion. That's not a mistake.
I just wanted to tell you about the prioritization policy.
I didn't mean to criticize you.

Recently, I'm wondering if it is worth adding translations for World & Commission Quest names. According to my analysis, users don't search those quest names so much.

Since there are so many World & Commission Quests and it is so tough to add them all. However, users are not so interested in them. So I think it should be less prioritized to translate them.

**Review by xicri — APPROVED** [review]

**Comment by Bill-Haku** [comment]
Well, it is just you made me find a mistake. There's no thing for you to apologize for it.

> Recently, I'm wondering if it is worth adding translations for World & Commission Quest names. According to my analysis, users don't search those quest names so much.
>
> Since there are so many World & Commission Quests and it is so tough to add them all. However, users are not so interested in them. So I think it should be less prioritized to translate them

I think you are right. I think only Archon Quests may be enough useful for users. So I will not translate them temporarily.

**Pull request merged**